src/main/java/appeng/client/Point.java

```diff
@@ -12,6 +14,10 @@ public Point(int x, int y) {
         this.y = y;
     }
 
+    public static Point fromTopLeft(Rectangle2d bounds) {
```

Comment:
Would it make sense to add this for all four corners? Mostly with the idea of positioning them relative to the main screen? E.g. upgrades/view cells are always relative to the `topRight`, network tool would be `bottomRight` and so on.
When changing something, this would avoid having to update all related widgets on top.

Reply:
Adding it makes no sense right now because it wouldn't be used :D